app/src/main/java/com/example/recipeopedia/fragments/FavoriteRecipesFragment.java

```diff
@@ -0,0 +1,76 @@
+package com.example.recipeopedia.fragments;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.recipeopedia.FavoriteRecipeAdapter;
+import com.example.recipeopedia.R;
+import com.example.recipeopedia.models.FavoriteRecipe;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class FavoriteRecipesFragment extends Fragment {
+    public static final String TAG = "FavoriteRecipesFragment";
+    private RecyclerView rvFavoriteRecipes;
+    protected FavoriteRecipeAdapter favoriteRecipeAdapter;
+    protected List<FavoriteRecipe> mFavoriteRecipes;
```

Comment:
this can be part of local scope, you don't need to store a reference to it in the fragment